AnkiDroid/src/main/java/com/ichi2/anki/noteeditor/Toolbar.java

```diff
@@ -206,7 +206,7 @@ public AppCompatImageButton insertItem(@IdRes int id, Drawable drawable, Runnabl
         // Hack - items are truncated from the scrollview
         View v = findViewById(R.id.editor_toolbar_internal);
 
-        int expectedWidth = getVisibleItemCount() * dpToPixels(32);
+        int expectedWidth = getVisibleItemCount() * dpToPixels(56);
```

Comment:
Extra credit if you extract these to variables (at least the padding)
```suggestion
        int expectedWidth = getVisibleItemCount() * dpToPixels(48 + 8); // width + 4dp padding on both sides
```